staging/src/k8s.io/client-go/tools/watch/retrywatcher.go

```diff
@@ -116,24 +116,24 @@ func (rw *RetryWatcher) doReceive() (bool, time.Duration) {
 		return false, 0
 
 	case io.ErrUnexpectedEOF:
-		klog.V(1).Infof("Watch closed with unexpected EOF: %v", err)
+		klog.V(1).ErrorS(err, "Watch closed with unexpected EOF")
```

Comment:
We should keep it to be InfoS.

Reply:
Done for all @caesarxuchao 
Thanks for comments.